tutorial/unity_local_realm.md

```diff
@@ -0,0 +1,608 @@
+
+The last step to create a new board is to update the `GameState` to make use of the new `PieceSpawner` and the `PieceEntity` that we just created.
+
+We'll go through these changes step by step. First we also need to import Realm here as well:
+
+```c#
+using Realms;
+```
+
+Then we add a private field to save our `Realm` instance to avoid creating it over and over again. We also create another private field to save the collection of pieces that are on the board and a notification token which we need for above mentioned notifications:
+
+```c#
+private Realm realm;
+private IQueryable<PieceEntity> pieceEntities;
+private IDisposable notificationToken;
+```
+
+In `Awake` we do need to get access to the `Realm`. This is achieved by opening an instance of it (1) and then asking it for all `PieceEntity` objects currently saved using `realm.All` (2) and assigning them to our `pieceEntities` field:
+
+```c#
+private void Awake()
+    {
+        realm = Realm.GetInstance(); // 1
+        pieceEntities = realm.All<PieceEntity>(); // 2
+
+        // 3
+        notificationToken = pieceEntities.SubscribeForNotifications((sender, changes, error) =>
+        {
+            // 4
+            if (error != null)
+            {
+                Debug.Log(error.ToString());
+                return;
+            }
+
+            // 5
+            // Initial notification
+            if (changes == null)
+            {
+                // Check if we actually have `PieceEntity` objects in our Realm (which means we resume a game).
+                if (sender.Count > 0)
+                {
+                    // 6
+                    // Each `RealmObject` needs a corresponding `GameObject` to represent it.
+                    foreach (PieceEntity pieceEntity in sender)
+                    {
+                        pieceSpawner.SpawnPiece(pieceEntity, pieces);
+                    }
+                }
+                else
+                {
+                    // 7
+                    // No game was saved, create a new board.
+                    pieceSpawner.CreateNewBoard(realm);
+                }
+                return;
+            }
+
+            // 8
+            foreach (var index in changes.InsertedIndices)
+            {
+                var pieceEntity = sender[index];
+                pieceSpawner.SpawnPiece(pieceEntity, pieces);
+            }
+        });
+    }
+```
+
+Note that collections are live objects. Meaning they will only be evaluated when we need them (i.e. when we access them) and they will always be re-evaluated in case they change. We also get notifications for those changes if we subscribed to them. This can be done by calling `SubscribeForNotifications` on a collection (3).
```

Comment:
> Note that collections are live objects. Meaning they will only be evaluated when we need them (i.e. when we access them) and they will always be re-evaluated in case they change  

I understand what you mean, but I think the sentence should be slightly rephrased. Something along the lines of
Note that collections are live objects. This has two positive implications:
1. Every access to the _object reference_ is always returning an updated representation of such object.
2. Because of 1., every subsequent change to the object will be visible any time that the object is referenced again.

I'm sure you can rephrase it better than I did, but my point was that the way you wrote seem to suggest that "live object" specifically means "lazy loaded" and a "auto re-evaluated". Those are more outcome of the memory mapping that goes on behind the scene.
@nirinchev will surely have better words than mine or correct my statement where incorrect.

Reply:
I got your point. I've added and re-phrased it. Let me know if it fits better.